src/App.css

```diff
@@ -1,10 +1,25 @@
+@import url('https://fonts.googleapis.com/css2?family=Anek+Devanagari:wght@700&display=swap');
+
 :root {
   --light-grey: #d1d3d4;
   --darker-grey: #9b9b9b;
   --white: #fff;
   --green: #97bf11;
 }
 
+.no-display {
+  display: none;
+}
+
+* {
+  list-style: none;
+  text-decoration: none;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Anek Devanagari', sans-serif;
+}
+
 .sidebar-container {
```

Comment:
- [ ]  the right container is floating at the bottom of the page in the desktop view.

![image](https://user-images.githubusercontent.com/19716151/205132409-fb1728a9-b61f-439a-ac8f-4cf396c356a1.png)

![image](https://user-images.githubusercontent.com/19716151/205132498-c58363e2-fc72-4c66-8c15-7ced58d54c4a.png)